### ISSUE TITLE
Facts - Hardware - Ipv4Addrs - TypeError: Ipv4Addrs.process_data() missing 1 required positional argument: 'data'

### ISSUE THREAD

**Issue by joserdot** [Fizzadar/pyinfra#831]
## Describe the bug
The data argument is empty in code. Please review also the Ipv6Addrs Fact 

## To Reproduce

inventory.py
``` 
nodes = ["192.168.15.100"]
``` 

deploy.py
``` 
from pyinfra import host
from pyinfra.facts.hardware import Ipv4Addrs, NetworkDevices

config.SUDO=True

ips = host.get_fact(Ipv4Addrs)                       
``` 

Run:
```
pyinfra inventory.py deploy.py --data=ssh_user=vagrant
``` 

## Expected behavior
Dict with network interfaces and address

## Meta
+ Include output of `pyinfra --support`.
```
--> Support information:

    If you are having issues with pyinfra or wish to make feature requests, please
    check out the GitHub issues at https://github.com/Fizzadar/pyinfra/issues .
    When adding an issue, be sure to include the following:

    System: Linux
      Platform: Linux-5.17.11-300.fc36.x86_64-x86_64-with-glibc2.35
      Release: 5.17.11-300.fc36.x86_64
      Machine: x86_64
    pyinfra: v2.2
    Executable: /home/jose/.local/bin/pyinfra
    Python: 3.10.4 (CPython, GCC 12.0.1 20220308 (Red Hat 12.0.1-0))
``` 
+ How was pyinfra installed (source/pip)? pip

+ Consider including output with `-vv` and `--debug`.
[error_01.log](https://github.com/Fizzadar/pyinfra/files/8842225/error_01.log)
[error_02.log](https://github.com/Fizzadar/pyinfra/files/8842226/error_02.log)

Thank you very much and congratulations for the work.

**Comment by Fizzadar** [comment]
Thanks for reporting this @joserafaelterra, will investigate and get this fixed ASAP!

**Comment by Fizzadar** [comment]
This has now been fixed in v2.3 via https://github.com/Fizzadar/pyinfra/commit/88270e75b693c6936a62ff94826b49250ee7903c, apologies for the delay on this one. Note to self: need to test short facts.
